docs/resources/azure_express_route_circuits.md

```diff
@@ -0,0 +1,120 @@
+---
+title: About the azure_express_route_circuits Resource
+platform: azure
+---
+
+# azure_express_route_circuits
+
+Use the `azure_express_route_circuits` InSpec audit resource to test properties related to express_route_circuits for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/expressroute/express-route-circuits/list) for  properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+## Syntax
+
+An `azure_express_route_circuits` resource block returns all Azure express route circuits, either within a Resource Group (if provided)
```

Comment:
Is `resource_group` required? Will it test all circuits in a subscription with `describe azure_express_route_circuits do`?